UnityProject/Assets/Scripts/Clothing/BackPack/InteractableStorage.cs

```diff
@@ -449,6 +453,8 @@ public bool Interact(HandActivate interaction)
 			return true;
 		}
 
+		if(interaction.Performer.TryGetComponent<PlayerNetworkActions>(out var actions)) actions.CmdTriggerStorageTrap(slots, interaction.Performer);
```

Comment:
```suggestion
		interaction.PerformerPlayerScript.PlayerNetworkedActions.CmdTriggerStorageTrap(slots, interaction.Performer);
```
Think you can get PlayerNetworkedActions from playerscript easily